autoware_cmake/package.xml

```diff
@@ -11,6 +11,7 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
   <build_export_depend>ament_cmake_auto</build_export_depend>
+  <build_export_depend>ament_lint_auto</build_export_depend>
```

Comment:
This should be `<buildtool_export_depend>ament_lint_auto</buildtool_export_depend>`, in fact `ament_cmake_auto` should be as well.

Reply:
done.